### PR TITLE
CCD-22211: Add archived flag to list snapshots and bump api version

### PR DESCRIPTION
**Pull request by cooneyde**

### CODE REVIEW

**Review by katkielbasa — APPROVED** [review]
LGTM

File: ComputeClient/Compute.Client/ApiUris.cs [hunk 1]
```diff
@@ -18,6 +18,9 @@ public static class ApiUris
         /// <summary>	The MCP 2.10 prefix. </summary>
         public const string MCP2_10_PREFIX = "caas/2.10/";
```
Comment (trevorflanagan) [inline ComputeClient/Compute.Client/ApiUris.cs]: How come you didnt remove this? I would expect all functions to be bumped along to 2.11.

Comment (jamesoleary123) [inline ComputeClient/Compute.Client/ApiUris.cs]: Do we know if have a seperate task to bump all the API versions to _2_11_ at some stage for zoidberg?

Can't remember when it was done the last time

**Review by jamesoleary123 — APPROVED** [review]

File: ComputeClient/Compute.Client/ApiUris.cs (hunk 1 above)
Comment (cooneyde) [reply ComputeClient/Compute.Client/ApiUris.cs]: No, I'll go create one. Are all 2.10 apis being bumped to 2.11?

Comment (cooneyde) [reply ComputeClient/Compute.Client/ApiUris.cs]: Just updated the PR to use v2.11 of api

Comment (jamesoleary123) [reply ComputeClient/Compute.Client/ApiUris.cs]: yeah we should be bumping everything to 2.11 at some point. Think the last time around we did that closer to release date but might be better to do it early so we can release the compute API if this change is the only thing needed for Zoidberg. Be good to add a task for that if we don't have one already

Comment (cooneyde) [reply ComputeClient/Compute.Client/ApiUris.cs]: I've bumped the version in this PR as part of this commit https://github.com/jimantz/Compute.Api.Client/pull/17/commits/720e2f64ba67b90b3bc76b55dc7ac83444b25c41

**Review by trevorflanagan — APPROVED** [review]

**Pull request merged**